app/src/main/res/values/strings.xml

```diff
@@ -143,13 +143,13 @@ We engage our community by contributing to open source, collaborating with the g
     <string name="our_github_link">Our Github Link</string>
     <string name="github_repository">Github Repository</string>
     <string name="our_website">Our Website</string>
-    <string name="systers_open_source_slack">Systers Open Source Slack</string>
-    <string name="systers_open_source_website">Systers Open Source Website</string>
+    <string name="systers_open_source_zulip">AnitaB.org Zulip</string>
```

Comment:
```suggestion
    <string name="anitab_org_open_source_zulip">AnitaB.org Zulip</string>
```
Why not change this as well 🤔 when you do change this you may have to change this somewhere else in the kotlin code

Reply:
@isabelcosta Can I do this change in a separate PR (Actually I have already squashed commits and all :). Also maybe we can refactor all instances in our codebase for "Systers Open Source" to "AnitaB. org" not only just the zulip string resource. Let me know your thoughts.

Reply:
Are there a lot of references to Systers in the app to change? @jddeep

Reply:
ok I see, no problem